services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js

```diff
@@ -140,8 +132,23 @@ qx.Class.define("osparc.desktop.preferences.pages.GeneralPage", {
       const preferences = osparc.Preferences.getInstance();
       preferences.bind("jobConcurrencyLimit", jobConcurrencySpinner, "value");
       jobConcurrencySpinner.addListener("changeValue", e => this.self().patchPreference("jobConcurrencyLimit", jobConcurrencySpinner, e.getData()));
-      form.add(jobConcurrencySpinner, this.tr("Maximum concurrent jobs"));
+      form.add(jobConcurrencySpinner, this.tr("Maximum Number of Concurrent Jobs"));
       box.add(new qx.ui.form.renderer.Single(form));
+      return box;
+    },
+    __createUserPrivacySettings: function() {
+      const box = this._createSectionBox("Privacy settings");
+
+      const label = this._createHelpLabel(this.tr("Help us improve Sim4Life user experience"), "text-13-italic");
```

Comment:
text-13-italic isn't valid and causes issues:
Uncaught Error: Error in property font of class qx.ui.basic.Label in method $$setFontImpl with incoming value 'text-13-italic': Is invalid!
See services/static-webserver/client/source/class/osparc/theme/Font.js for available fonts and definitions

Reply:
OPS: my bad here, I have just deleted something I thought was extra. Put back the font style.